examples/pytorch/dimenet/README.md

```diff
@@ -0,0 +1,125 @@
+num_radial        int    Number of radial basis functions.                                      Default is 6   
+envelope_exponent int    Shape of the smooth cutoff.                                            Default is 5   
+cutoff            float  Cutoff distance for interatomic interactions.                          Default is 5.0 
+num_before_skip   int    Number of residual layers in interaction block before skip connection. Default is 1   
+num_after_skip    int    Number of residual layers in interaction block after skip connection.  Default is 2   
+num_dense_output  int    Number of dense layers for the output blocks.                          Default is 3   
+targets           list   List of targets to predict.                                            Default is ['mu']
+output_init       string Initial function name for output layer.                                Default is 'GlorotOrthogonal'
+```
+
+###### Training options
+```
+num_train         int   Number of training samples.                     Default is 110000
+num_valid         int   Number of validation samples.                   Default is 10000
+data_seed         int   Random seed.                                    Default is 42
+lr                float Learning rate.                                  Default is 0.001
+weight_decay      float Weight decay.                                   Default is 0.0001
+ema_decay         float EMA decay.                                      Default is 0.
+batch_size        int   Batch size.                                     Default is 100
+epochs            int   Training epochs.                                Default is 300
+early_stopping    int   Patient epochs to wait before early stopping.   Default is 20
+num_workers       int   Number of subprocesses to use for data loading. Default is 18
+gpu               int   GPU index.                                      Default is 0, using CUDA:0
+interval          int   Time intervals for model evaluation.            Default is 50
+step_size         int   Period of learning rate decay.                  Default is 100
+gamma             float Factor of learning rate decay.                  Default is 0.3
+```
+
+##### Examples
+
+The following commands learn a neural network and predict on the test set.
+Training a DimeNet model on QM9 dataset.
+```bash
+python main.py --model-cnf config/dimenet.yaml
+```
+Training a DimeNet++ model on QM9 dataset.
+```bash
+python main.py --model-cnf config/dimenet_pp.yaml
+```
+For faster experimentation, you should first put the author's `pretrained` folder here, and you can convert a TensorFlow model to PyTorch model by using the following commands.
+```
+python convert_tf_ckpt_to_pytorch.py --model-cnf config/dimenet_pp.yaml --convert-cnf config/convert.yaml
+```
+Then you can set `flag: True` in `dimenet_pp.yaml` and run the above script, DimeNet++ will use the pretrained weights to predict on the test set.
+
+### Performance
+
```

Comment:
I assume these are differences in experiment setting. Why did you use a different setting?

Reply:
Should I fix learning rate decay differences later?

Reply:
Did you simply do so for a better model performance?